docs/designers-developers/designers/menu-item.md

```diff
@@ -0,0 +1,117 @@
+# MenuItem
+
+`MenuItem` is a component which renders a button intended to be used in combination with the `MenuGroup` component.
+
+![An image of a MenuItem being highlighted inside of a DropdownMenu component](https://wordpress.org/gutenberg/files/2018/11/MenuItem.png)
+
+1. MenuItem
+
+
+
+## Table of contents
+
+
+
+1. [Design guidelines](http://#design-guidelines)
+2. [Development guidelines](http://#development-guidelines)
+3. [Related components](http://#related-components)
+
+
+
+## Design guidelines
+
+###Usage
+
+A `MenuGroup` contiaining `MenuItem`s can be used within a `Dropdown`. A `MenuGroup` can also have other `MenuGroup`s within it so menus can be nested.
+
+## Development guidelines
+
+###Usage
+
+`MenuItem` is a component which renders a button intended to be used in combination with the `MenuGroup` component.
+
+```
+import { MenuItem } from '@wordpress/components';
+import { withState } from '@wordpress/compose';
+```
```

Comment:
I think the import statement code block should be grouped with the below code block instead of being separate